mlir/include/mlir/Dialect/LLVMIR/LLVMAttrDefs.td

```diff
@@ -447,6 +465,39 @@ def LLVM_DILocalVariableAttr : LLVM_Attr<"DILocalVariable", "di_local_variable",
   let assemblyFormat = "`<` struct(params) `>`";
 }
 
+//===----------------------------------------------------------------------===//
+// DIGlobalVariableExpressionAttr
+//===----------------------------------------------------------------------===//
+
+def LLVM_DIGlobalVariableExpressionAttr : LLVM_Attr<"DIGlobalVariableExpression", "di_global_variable_expression",
```

Comment:
Nit: Can you constrain this to 80 characters per line, similar to what applies to the C++ files.
The same applies to the changes above.